notebooks/official/workbench/fraud_detection/fraud-detection-model.ipynb

```diff
@@ -74,7 +77,7 @@
         "\n",
         "This tutorial shows you how to build, deploy, and analyze predictions from a simple [Random Forest](https://en.wikipedia.org/wiki/Random_forest) model using tools like scikit-learn, Vertex AI, and the [What-IF Tool (WIT)](https://cloud.google.com/ai-platform/prediction/docs/using-what-if-tool) on a synthetic fraud transaction dataset to solve a financial fraud detection problem.\n",
         "\n",
-        "**Note:** The What-If tool widget used in this notebook only runs in a Colab environment. It is not explicitly supported for Vertex AI user-managed notebook instances. \n",
+        "**Note:** The What-If tool widget used in this notebook only runs in a Colab environment. It isn't explicitly supported for Vertex AI user-managed notebook instances. \n",
```

Comment:
"notebook" should be plural here "Vertex AI user-managed notebooks instances" (according to Aaron Dietz)